src/exampleTypes/ResultTable.js

```diff
@@ -59,44 +62,44 @@ let HighlightedColumnHeader = observer(
       <Cell key="additionalFields">Other Matches</Cell>
     ) : null
 )
-HighlightedColumnHeader.displayName = 'HighlightedColumnHeader'
 
 let labelForField = (schema, field) =>
   _.getOr(field, 'label', _.find({ field }, schema))
 
-let HighlightedColumn = observer(
+let HighlightedColumn = _.flow(
+  setDisplayName('HighlightedColumn'),
+  observer,
+  withTheme
+)(
   ({
     node,
     results = _.result('slice', getResults(node)),
     additionalFields = _.result('0.additionalFields.slice', results),
-    Cell = 'td',
-    Table = 'table',
-    Modal = null,
     schema,
+    Cell = 'td',
+    theme: { Modal, Table },
```

Comment:
Remember this might be overriden on a per instance basis (e.g. Ilian's current use case)